test/step-definitions/requests.ts

```diff
@@ -19,14 +19,16 @@ export class requests {
 
   @before()
   public async before(): Promise<void> {
-    const options: cliOptions = {
-      namespaceUrl: 'http://localhost:8082/v2/schemas/namespaces',
-      authUrl: process.env.STARGATE_AUTH_URL,
-      keyspace: 'eddiehub',
-      password: 'cassandra',
-      username: 'cassandra',
-    };
-    await setup(options);
+    if (process.env.STARGATE_BASEURL) {
```

Comment:
I think this resolves the problem

Reply:
`options` will be undefined, I think we will need to pass in other options for Astra in the cloud

Reply:
```ts
✗ npm run test:e2e 

> api@0.5.2 test:e2e /Users/ej/Downloads/repos/api
> cucumber-js -p default

(node:6148) UnhandledPromiseRejectionWarning: Error: please define keyspace name
    at Object.<anonymous> (/Users/ej/Downloads/repos/api/node_modules/@cahllagerfeld/setup-stargate/src/setup.ts:9:11)
    at step (/Users/ej/Downloads/repos/api/node_modules/@cahllagerfeld/setup-stargate/dist/setup.js:33:23)
    at Object.next (/Users/ej/Downloads/repos/api/node_modules/@cahllagerfeld/setup-stargate/dist/setup.js:14:53)
    at /Users/ej/Downloads/repos/api/node_modules/@cahllagerfeld/setup-stargate/dist/setup.js:8:71
```

Reply:
When doing it with the online DB we dont need any configuration.
```ts
 authUrl: process.env.STARGATE_AUTH_URL
          ? process.env.STARGATE_AUTH_URL
          : '',
```
Would something like this help`?

Reply:
I will take a closer look Sunday morning

Reply:
Actually the error is ...

```ts
               _                                   _                                    _          
  ___    ___  | |_   _   _   _ __            ___  | |_    __ _   _ __    __ _    __ _  | |_    ___ 
 / __|  / _ \ | __| | | | | | '_ \   _____  / __| | __|  / _` | | '__|  / _` |  / _` | | __|  / _ \
 \__ \ |  __/ | |_  | |_| | | |_) | |_____| \__ \ | |_  | (_| | | |    | (_| | | (_| | | |_  |  __/
 |___/  \___|  \__|  \__,_| | .__/          |___/  \__|  \__,_| |_|     \__, |  \__,_|  \__|  \___|
                            |_|                                         |___/                      
couldn't create stargate namespace
    ✖ failed
      Error: Request Failed: undefined
      Stack Trace: connect ECONNREFUSED 127.0.0.1:8081
          at axiosRequest (/Users/ej/Downloads/repos/api/node_modules/@astrajs/rest/src/rest.js:126:11)
          at processTicksAndRejections (internal/process/task_queues.js:93:5)
          at Object.createClient (/Users/ej/Downloads/repos/api/node_modules/@astrajs/rest/src/rest.js:63:24)
          at Object.createClient (/Users/ej/Downloads/repos/api/node_modules/@astrajs/collections/src/collections.js:146:22)
```

Reply:
Fresh pair of eyes in the morning helped. I got the tests passing by needing the namespace/keyspace on the cloud.

Reply:
But I also had to comment out the `import` not sure why it is being triggered

```
import { setup, cliOptions } from '@cahllagerfeld/setup-stargate';
```